source/braille.py

```diff
@@ -373,15 +374,20 @@
 
 
 def NVDAObjectHasUsefulText(obj: "NVDAObject") -> bool:
+	"""Does obj contain useful text to display in braille
+
+	:param obj: object to check
+	:return: True if there is useful text, False if not
+	"""
 	if objectBelowLockScreenAndWindowsIsLocked(obj):
 		return False
 	import displayModel
 	if issubclass(obj.TextInfo,displayModel.DisplayModelTextInfo):
 		# #1711: Flat review (using displayModel) should always be presented on the braille display
 		return True
-	else:
-		# Let the NVDAObject choose if the text should be presented
-		return obj._hasNavigableText
+	if obj._hasNavigableText or isinstance(obj, EditableText):
+		return True
+	return False

```

Comment:
these changes should probably be removed in the diff - I imagine merging master in will fix this